RevenueCatUI/CustomerCenter/Data/SubscriptionInformation.swift

```diff
@@ -20,11 +20,19 @@ struct SubscriptionInformation {
     let title: String
     let durationTitle: String
     let price: String
-    let nextRenewalString: String?
+    let expirationDateString: String?
     let productIdentifier: String
 
-    var renewalString: String {
-        return active ? (willRenew ? "Renews" : "Expires") : "Expired"
+    var expirationString: String {
+        return active ? (willRenew ? "Next billing date" : "Expires") : "Expired"
+    }
+
+    var explanation: String {
+        return active ? (
+            willRenew ?
+                "This is your subscription with the soonest billing date." :
+                "This is your subscription with the soonest expiration date."
```

Comment:
Maybe double-check but I think these should be `earliest`?
```suggestion
                "This is your subscription with the earliest billing date." :
                "This is your subscription with the earliest expiration date."
```

Reply:
Yes! I already didn't like "soonest" haha, and also "the earliest" is [more common](https://books.google.com/ngrams/graph?content=the+soonest%2Cthe+earliest&year_start=1900&year_end=2019&corpus=en-2019&smoothing=20).